src/ui/Startup.cs

```diff
@@ -64,13 +64,18 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     apm.ApplicationParts.Remove(item);
                 }
+            }).AddCookieTempDataProvider(options => {
```

Comment:
We don't want this for dev so it should go into the Configure() method with something along these lines: Cookie.SecurePolicy = hostingEnvironment.IsDevelopment()
            ? CookieSecurePolicy.SameAsRequest
            : CookieSecurePolicy.Always